### DISCUSSION TITLE
pi pico W VBUS detection

### DISCUSSION THREAD

**Discussion by JuanZubi** [micropython/micropython#9723] [Using MicroPython]
Can´t find how to know the presence of 5 V from USB in pico W.
In previous pi pico was very easy to detect the VBUS presence, just reading GPIO24.
But in pi pico W this pin is used and the datasheet informs that VBUS state can be kown using WL_GPIO02.
Can´t find a way, in Micropython, to access to that pin.
Can anyone help me to find it out

**Comment by jimmo** [comment]
@JuanZubi 
I haven't tested this, but I think you can use `pin = machine.Pin("WL_GPIO2")`

**Reply by JuanZubi** [reply]
Great, it works!!!!!
My mistake was that I was using "WL_GPIO02" instead of "WL_GPIO2". By error I inserted an extra"0"
Thank you very much, jimmo.
